man/man8/zfs.8

```diff
@@ -3304,16 +3304,24 @@ Displays all ZFS file systems currently mounted.
 .Op Fl o Ar options
 .Fl a | Ar filesystem
 .Xc
-Mounts ZFS file systems.
+Mount ZFS filesystem on its configured
+.Sy mountpoint
```

Comment:
because 'mountpoint' is a generic term, I think we should be specific here, that we are talking about the mountpoint property.